playbook/app/pb_kits/playbook/pb_flex/_flex.scss

```diff
@@ -69,13 +69,15 @@ $gapTypes: (
     justify-content: center;
   }
 
+
   // Alignment: Align Items
-  &[class*=align_items_left],
-  &[class*=align_items_top],
-  &[class*=align_items_start] {
+  &[class*=justify_items_left],
```

Comment:
@JelenaD1 let's change this back to align_items

Reply:
I've made the changes but existing styling is still breaking on the page above. Any idea what to inspect next?